src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Functions.ArtifactAssembler
+{
+    internal sealed class ArtifactAssembler
+    {
+        private const string StagingDirName = "staging";
+        private const string InProc8DirectoryName = "in-proc8";
+        private const string InProc6DirectoryName = "in-proc6";
+
+        /// <summary>
+        /// The artifacts for which we want to pack a custom host with it.
+        /// This dictionary contains the artifact name and the corresponding runtime identifier value.
+        /// </summary>
+        private readonly Dictionary<string, string> _customHostArtifacts = new()
+        {
+            { "Azure.Functions.Cli.min.win-x64", "win-x64" },
+    //        { "Azure.Functions.Cli.min.win-arm64", "win-arm64" }
```

Comment:
should this be commented out?

Reply:
Removed. Commented out for local testing